tests/spread/amd64/classic/task.yaml

```diff
@@ -0,0 +1,52 @@
+summary: Build classic Ubuntu server image
+
+kill-timeout: 3h
+
+execute: |
+    mkdir -p /tmp/${SPREAD_VARIANT}/mount
+    pushd ${SPREAD_VARIANT}
+
+    imagecraft --verbosity debug pack --destructive-mode
+
+    IMG_NAME=pc.img
```

Comment:
I guess this is unused right now? Or did you mean on using this instead of `pc.img` directly?

Reply:
Correct, I meant on using this var. I will fix it. It will need to be reworked when we are able to better control the artifacts. Here I am lucky that we create a img called `pc.img` in both cases.